src/utils/account/account.service.ts

```diff
@@ -196,10 +205,10 @@ export class AccountService {
     try {
       res = await this.msGraphService.resetPassword(accountUpn, newPassword);
     } catch (error) {
-      this.logger.error(
-        `Failed to reset password for account ${accountUpn}`,
-        error.message,
-        LogContext.COMMUNITY
+      throw new AccountException(
+        `Failed to reset password for account ${accountUpn} ${error}`,
+        LogContext.COMMUNITY,
+        CherrytwistErrorStatus.ACCOUNT_DELETION_FAILED
```

Comment:
Looks like the wrong code

Reply:
Fixed in bfa0a20312bb781d6ffc7afde654580cc28b6159